console/network/src/mainnet_v0.rs

```diff
@@ -134,19 +134,14 @@ impl Network for MainnetV0 {
     /// The transmission checksum type.
     type TransmissionChecksum = u128;
 
-    /// The block height from which consensus V2 rules apply.
+    /// The block heights at which consensus versions are updated.
```

Comment:
```suggestion
    /// A list of (block_height, consensus_version) pairs indicating when each consensus version takes effect.
```

Documentation should explain what the tuple values are.

Reply:
4d3b948e0